src/frontends/onnx/frontend/src/op/greater_or_equal.cpp

```diff
@@ -28,6 +28,46 @@ ov::OutputVector greater_or_equal(const Node& node) {
 }
 }  // namespace set_1
 
+
+namespace onnx_import {
+namespace op {
+
+OutputVector relu(const Node& node) {
+    const auto opset_version = node.get_opset_version();
+
+    if (opset_version >= 1 && opset_version <= 6) {
```

Comment:
No need to do this here, it is done in ops_bridge.cpp logic when it is looking for correct implementation.

But maybe reasonable to re-use this approach for checking input types, depends on opset version. Could you try?